docs/design/cincinnati.md

```diff
@@ -49,14 +49,24 @@ An example of this metadata document can be seen here:
 }
 ```
 
-A simple example of this directed acyclic graph can be seen in Figure 1. In this example, four versions were released (`1.0.0`, `1.1.0`, `1.2.0`, and `1.3.0`). After releasing `1.1.0`, it is discovered that there is a critical bug and this version is subsequently removed from the graph. Because of this, it will not be possible for clients who are currently running `1.0.0` to update to `1.2.0`.
+A simple example of this directed acyclic graph can be seen in Figure 1. In this example, four versions were released (`1.0.0`, `1.1.0`, `1.2.0`, and `1.3.0`).
 
 <figure align="center">
-  <img src="figures/cincinnati-1.svg" alt="Figure 1: A visualized DAG with three groups and a broken release" />
-  <figcaption>Figure 1: A visualized DAG with three groups and a broken release</figcaption>
+  <img src="figures/cincinnati-initial.svg" alt="Figure 1: A visualized DAG with four releases" />
+  <figcaption>Figure 2: A visualized DAG with four releases</figcaption>
 </figure>
 
-In order to allow the transition mentioned above, a new version, `1.1.1`, needs to be released. In order to insert itself into the graph, it will need to make use of both the previous and next fields in its metadata:
+After releasing `1.1.0`, it is discovered that there is a critical bug and this version is subsequently flagged as degraded.
+Edges which lead into the degraded release are removed to prevent clients from upgrading to the degraded release.
```

Comment:
I would try not to mix different levels of abstractions here. "Edge" is a graph term and "release" is an application term. I suggest to either replace "edge" by "update path" or "release" by "node".

Reply:
Fixed in b9cea22 -> 1221d80.